__tests__/app.test.js

```diff
@@ -94,40 +96,40 @@ describe("GET /api/articles/:article_id", () => {
       .get("/api/articles/bananas")
       .expect(400)
       .then(({ body }) => {
-        expect(body.msg).toBe('Bad request')
+        expect(body.msg).toBe("Bad request");
       });
   });
   test("Bad request- valid but out of range- returns 404 object not found", () => {
     return request(app)
       .get("/api/articles/900000")
       .expect(404)
       .then(({ body }) => {
-        expect(body.msg).toBe('Object not found')
-        });
-    });
-})
+        expect(body.msg).toBe("Object not found");
+      });
+  });
+});
 
 describe("PATCH /api/articles/:article_id", () => {
   test("takes an object and returns the updated article object", () => {
     const input = { inc_votes: 3 };
     const originArticle1 = {
-        article_id: 1,
-        title: 'Living in the shadow of a great man',
-        topic: 'mitch',
-        author: 'butter_bridge',
-        body: 'I find this existence challenging',
-        created_at: '2020-07-09T20:11:00.000Z',
-        votes: 100
-      }
+      article_id: 1,
+      title: "Living in the shadow of a great man",
+      topic: "mitch",
+      author: "butter_bridge",
+      body: "I find this existence challenging",
+      created_at: "2020-07-09T20:11:00.000Z",
+      votes: 100,
+    };
     return request(app)
       .patch("/api/articles/1")
       .send(input)
       .expect(200)
       .then(({ body }) => {
         expect(typeof body.updatedArticle).toBe("object");
-        expect(Array.isArray(body.updatedArticle)).toBe(false)
-        expect(body.updatedArticle).toBeInstanceOf(Object)
-        expect(body.updatedArticle).not.toEqual(originArticle1)
+        expect(Array.isArray(body.updatedArticle)).toBe(false);
+        expect(body.updatedArticle).toBeInstanceOf(Object);
```

Comment:
You have some extras here that you may not need (a similar case above)
You are asserting that the item is NOT an array, ok. You then assert that it is an instance of an object. An array's type is object, but it will never BE an instance of an object. Line 131 is making line 130 redundant.

Nothing really wrong, but it is not needed.